### ISSUE TITLE
Dropdown component is not focusable with keyboard only

### ISSUE THREAD

**Issue by laurentC35** [InseeFr/Lunatic#70]
You can the the issue in Lunatic editor : https://inseefr.github.io/Lunatic/editor/
